Signal/src/ViewControllers/ConversationView/Cells/ConversationMediaView.swift

```diff
@@ -7,6 +7,12 @@ import Foundation
 @objc(OWSConversationMediaView)
 public class ConversationMediaView: UIView {
 
+    private enum MediaError {
+        case missing
+        case invalid
+        case failed
```

Comment:
We need to distinguish these cases more carefully.  In particular, we want to avoid "flashing" the "invalid media indicator" when media start/complete downloading.